### ISSUE TITLE
The evaluation results vary greatly across different scenes

### ISSUE THREAD

**Issue by zlm200** [gemcollector/RL-ViGen#25]
I strictly followed the pieg_config to train the Sawyer robot in robosuite for the task of block grasping. The only possible modification was that I did not use EGL (as my system seemed to have issues initializing the DIR2 page); instead, I used OSMesa for rendering. I believe this change has minimal impact. After 2000 episodes of training, I selected weights that had reached a score of 470 during training to carry out the evaluation. I changed the mode to eval-easy in the robo_config. I found that there is a significant difference between different scenes, and the average reward after training is just over 100, which is not as high as the 305 shown in figure 19 of the paper. I am not sure if there is an issue with one of my training steps. Could you please provide me with some advice?
[file_5.txt](https://github.com/user-attachments/files/18258660/file_5.txt)
[eval.csv](https://github.com/user-attachments/files/18258664/eval.csv)
[train.csv](https://github.com/user-attachments/files/18258665/train.csv)

**Comment by gemcollector** [comment]
Hi, there. There are some details I should mention:
- Our experiments use the franka arm.
- The results in Figure 19 are the **aggregate results** . If you want to check the score of each task, you should look the `results/evaluation_score.csv`. So for the `Lift` task, I think the Sawyer robot and your score are similar. So the results make sense.

**Comment by zlm200** [comment]
okay,I understand, thank you for your reply.
